crates/router/src/connector/forte/transformers.rs

```diff
@@ -0,0 +1,263 @@
+    pub status: String,
+    pub action: String,
+    #[serde(rename = "authorization_amount")]
+    pub authorization_amount: i64,
+    #[serde(rename = "authorization_code")]
+    pub authorization_code: String,
+    #[serde(rename = "received_date")]
+    pub received_date: String,
+    #[serde(rename = "billing_address")]
+    pub billing_address: BillingAddress,
+    pub card: Card,
+    pub response: Response,
+    pub links: Links,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct PhysicalAddress {
+    #[serde(rename = "street_line1")]
+    pub street_line1: String,
+    #[serde(rename = "street_line2")]
+    pub street_line2: String,
+    pub locality: String,
+    pub region: String,
+    pub country: String,
+    #[serde(rename = "postal_code")]
+    pub postal_code: String,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Links {
+    pub disputes: String,
+    pub settlements: String,
+    #[serde(rename = "self")]
+    pub self_field: String,
+}
+
+//TransactionId Types end
+//Types End
+impl TryFrom<&types::PaymentsAuthorizeRouterData> for FortePaymentsRequest  {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &types::PaymentsAuthorizeRouterData) -> Result<Self,Self::Error> {
+
+        match item.request.payment_method_data {
+            api::PaymentMethod::Card(ref ccard) => {
+                let request  =  FortePaymentsRequest{
+                    billing_address : BillingAddress { first_name: ccard.card_holder_name.peek().clone(), last_name: ccard.card_holder_name.peek().clone() },
+                    card: Card {
+                        card_type               : String::from("visa"),
+                        name_on_card            : ccard.card_holder_name.peek().clone(),
+                        account_number          : ccard.card_number.peek().clone(),
+                        expire_month            : ccard.card_exp_month.peek().clone(),
+                        expire_year             : ccard.card_exp_year.peek().clone(),
+                        card_verification_value : ccard.card_cvc.peek().clone(),
+                    },
+                    authorization_amount: item.request.amount as f64,
+                    subtotal_amount: item.request.amount as f64,
+                };
+                Ok(request)
+            }
+            _ => Err(
+                errors::ConnectorError::NotImplemented("Current Payment Method".to_string()).into(),
+            ),
+    }
+}
+}
+
+//TODO: Fill the struct with respective fields
+// Auth Struct
+pub struct ForteAuthType {
+    pub(super) api_key: String
+}
+
+impl TryFrom<&types::ConnectorAuthType> for ForteAuthType  {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(_auth_type: &types::ConnectorAuthType) -> Result<Self, Self::Error> {
+        todo!()
```

Comment:
handle the auth_type conversion logic in this try block

Reply:
done.